dGame/dComponents/InventoryComponent.cpp

```diff
@@ -1019,6 +1052,14 @@ void InventoryComponent::EquipItem(Item* item, const bool skipChecks)
 
 void InventoryComponent::UnEquipItem(Item* item)
 {
+	// if (item->GetLot() == 8092 && m_Parent->GetGMLevel() >= GAME_MASTER_LEVEL_OPERATOR && hasCarEquipped == true)
```

Comment:
I would just delete this code, will likely never be used.

Reply:
Ah yeah, was going to try to use this function at some point but forgot to deprecate it